app.js

```diff
@@ -1,7 +1,7 @@
 console.log("js is linked");
 
 $(document).ready(function() {
-  console.log("jquery is linked");
```

Comment:
JQuery is linked in the HTML with the CDN link. This just says the document has finished loading.